db/migrate/20240702224053_add_personal_openai_api_key_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPersonalOpenaiApiKeyToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :personal_openai_api_key, :string
```

Comment:
I think calling it `personal_openai_api_key` may be to specific, I think `openai_api_key` would be enough, since it already is tied to the user